util/network-devp2p/Cargo.toml

```diff
@@ -10,6 +10,8 @@ edition = "2018"
 [dependencies]
 ansi_term = "0.11"
 bytes = "0.4"
+derive_more = "0.99"
+enr = { version = "0.1.0-alpha.4", default-features = false, features = ["rust-secp256k1"] }
```

Comment:
Is this new version something you've contributed to? Is there a PR pending?

Reply:
My contributions have been merged: https://github.com/AgeManning/enr/pull/6

Reply:
…so we're awaiting the release then?

Reply:
They became part of `0.1.0-alpha.4` :-)

Reply:
So waiting for 0.1.0 then.

Reply:
any news on this?

Reply:
I think it's fine to merge this PR with `0.1.0-alpha.4` or `0.1.0-alpha.5` and update it later.

Reply:
I disagree, but if it's going to take them a long while before releasing, then maybe I have to swallow it.

Reply:
I bumped enr to 0.1.0-alpha.5 as it contains the patches requested here. Any further updates to that crate should not break code paths used by OpenEthereum.